src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php

```diff
@@ -3,20 +3,14 @@
 namespace Pelagos\Bundle\AppBundle\Rabbit\Consumer;
 
 use Doctrine\ORM\EntityManager;
-
 use OldSound\RabbitMqBundle\RabbitMq\ConsumerInterface;
-
-use PhpAmqpLib\Message\AMQPMessage;
-
-use Symfony\Bridge\Monolog\Logger;
-
 use Pelagos\Entity\Dataset;
```

Comment:
Normally Pelagos uses are last, also leave a line between each group. 3rd party bundles and our own use statements.

Reply:
Changes made.